plugins/brimcap/brimcap-plugin.ts

```diff
@@ -329,21 +331,43 @@ export default class BrimcapPlugin {
     this.api.configs.add(brimcapConfig)
   }
 
+  private async updateSuricata() {
+    const {zdepsDirectory} = this.api.getAppConfig()
+    const cmdName =
+      process.platform === "win32" ? "suricataupdater.exe" : "suricataupdater"
+    const cmdPath = path.join(zdepsDirectory, "suricata", cmdName)
```

Comment:
I hope to consolidate code like this in `app/ipc/paths`, and expose it through the plugin api.

Maybe like `this.api.getZdepPath("suricata/suricataupdater")`. It could be responsible for adding the exe to the end of commands.

Reply:
Ah yeah that makes sense